package.json

```diff
@@ -14,6 +14,7 @@
     "near-ca": "^0.0.9"
   },
   "devDependencies": {
+    "@types/node": "^20.14.2",
```

Comment:
This was a missing peer-dependency

Reply:
You can get rid of the `/// <reference>` comment at the top of the TS file with this probably.